changelog/13402.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
```

Comment:
I am not sure if the change can be accounted for a bug. Generally, for doc changes like this, no changelog is needed.

Reply:
Okay, I removed the changelog file.  The contributing doc mentioned that no issue was needed, but I wasn't sure about the changelog.